plugins/modules/volume.py

```diff
@@ -176,6 +176,33 @@ def create(self, resource):
         self._module.fail_on_missing_params(['name', 'size_gb'])
         return super(AnsibleCloudscaleVolume, self).create(resource)
 
+    def find_difference(self, key, resource, param):
+        is_different = False
+
+        if key == 'servers':
+            server_has = resource[key]
+            server_wanted = param
+            if len(param) > len(server_has):
```

Comment:
Any reason why you check for "greather than" only? I would assume there is a difference whenever there's a difference in length.

Reply:
Also, if you already set `server_wanted` to be equal to `param`, which I think is reasonable, you should refer to it here using `server_wanted` and not `param`.